src/NexusMods.App.UI/Pages/LoadoutPage/LoadoutComponents.cs

```diff
@@ -85,21 +86,28 @@ protected override void Dispose(bool disposing)
             base.Dispose(disposing);
         }
     }
+
+    public sealed class Locked : ReactiveR3Object, IItemModelComponent<Locked>, IComparable<Locked>
```

Comment:
I would rename to `LockedComponent` for clarity. 
Same thing with `IsEnabled` -> `IsEnabledComponent` tbh

Reply:
These are child classes of `LoadoutComponents`. If you want to access them, you always have to do `LoadoutComponents.IsEnabled`. Adding the `Component` suffix is somewhat redundant as you'd have `LoadoutComponents.IsEnabledComponent`.

Reply:
It would still be clearer, it isn't blowing the name out